py3status/i3status.py

```diff
@@ -143,6 +145,25 @@ def i3status_config_reader(self, i3status_config_path):
                 in_section = True
                 section_name = 'order'
 
+            if (not in_section) and line.startswith('group'):
```

Comment:
Sure

Really I'd like to change to pyparsing or something but that would be for another day